scipy/stats/_continuous_distns.py

```diff
@@ -6384,6 +6384,38 @@ def _munp(self, n, c):
     def _entropy(self, c):
         return np.log(2.0/c) + 1.0
 
+    @_call_super_mom
+    def fit(self, data, *args, **kwds):
+        data, fc, floc, fscale = _check_fit_input_parameters(self, data,
+                                                             args, kwds)
+
+        # Specialize MLE only when location is known.
+        if floc is None:
+            return super(type(self), self).fit(data, *args, **kwds)
+
+        # Raise an error if any observation has zero likelihood.
+        if np.any(data <= floc):
+            raise FitDataError("loglaplace", lower=floc, upper=np.inf)
+
+        # Remove location from data.
+        if floc != 0:
+            data = data - floc
+
+        # When location is zero, the log-Laplace distribution is related to
+        # the Laplace distribution in that if X ~ Laplace(loc=a, scale=b),
+        # then Y = exp(X) ~ LogLaplace(c=1/b, loc=0, scale=exp(a)).  It can
+        # be shown that the MLE for Y is the same as the MLE for X = ln(Y).
+        # Therefore, we reuse the formulas from laplace.fit() and transform
+        # the result back into log-laplace's parameter space.
+        a, b = laplace.fit(np.log(data),
+                           floc=np.log(fscale) if fscale is not None else None,
+                           fscale=1/fc if fc is not None else None,
+                           method='mle')
+        if fscale is None:
+            fscale = np.exp(a)
+        if fc is None:
+            fc = 1 / b
+        return fc, floc, fscale
```

Comment:
There has been some confusion in the past about `fscale` vs `scale`, etc., so I try to reserve `f`-prefixed variable names for values that are fixed.

```suggestion
        loc = floc
        scale = np.exp(a) if fscale is None else fscale
        c = 1/b if fc is None else fc
        return c, loc, scale
```